plaso/cli/image_export_tool.py

```diff
@@ -297,8 +299,12 @@ def _ExtractWithFilter(
       source_path_specs (list[dfvfs.PathSpec]): path specifications to extract.
       destination_path (str): path where the extracted files should be stored.
       output_writer (CLIOutputWriter): output writer.
-      filter_file_path (str): path of the file that contains the filter
-          expressions.
+      artifacts_registry (artifacts.ArtifactDefinitionsRegistry]): artifact
```

Comment:
remove the `]` in `(artifacts.ArtifactDefinitionsRegistry])`

Reply:
Done